sound/soc/intel/boards/Kconfig

```diff
@@ -593,22 +593,6 @@ config SND_SOC_INTEL_CML_LP_DA7219_MAX98357A_MACH
 	   Say Y or m if you have such a device. This is a recommended option.
 	   If unsure select "N".
 
-config SND_SOC_INTEL_SOF_CML_RT1011_RT5682_MACH
-	tristate "CML with RT1011 and RT5682 in I2S Mode"
-	depends on I2C && ACPI
-	depends on MFD_INTEL_LPSS || COMPILE_TEST
-	depends on SND_HDA_CODEC_HDMI && SND_SOC_SOF_HDA_AUDIO_CODEC
-	select SND_SOC_RT1011
-	select SND_SOC_RT5682_I2C
-	select SND_SOC_DMIC
-	select SND_SOC_HDAC_HDMI
-	select SND_SOC_INTEL_HDA_DSP_COMMON
-	help
-	  This adds support for ASoC machine driver for SOF platform with
-	  RT1011 + RT5682 I2S codec.
-	  Say Y if you have such a device.
-	  If unsure select "N".
-
```

Comment:
if a .config file only selected this option, then with a make olddefconfig the sof_rt5682 machine would not be selected.
For true backwards compatibility I think we need to keep this Kconfig and use 'imply sof_rt5682' or something to keep the link.

Reply:
will add the option back